geometric/internal.py

```diff
@@ -1860,15 +1862,23 @@ def GInverse_SVD(self, xyz):
         V = VT.T
         UT = U.T
         Sinv = np.zeros_like(S)
+        Ssqrt = np.zeros_like(S)
         LargeVals = 0
         for ival, value in enumerate(S):
             # print "%.5e % .5e" % (ival,value)
             if np.abs(value) > 1e-6:
+                if sqrt: value = np.sqrt(value)
                 LargeVals += 1
                 Sinv[ival] = 1/value
+                Ssqrt[ival] = value
+
         # print "%i atoms; %i/%i singular values are > 1e-6" % (xyz.shape[0], LargeVals, len(S))
         Sinv = np.diag(Sinv)
         Inv = multi_dot([V, Sinv, UT])
+        if sqrt:
```

Comment:
Please add a comment here explaining when "sqrt" is used

Reply:
I added some [comments](https://github.com/leeping/geomeTRIC/pull/197/files#diff-e41d40e8871e4728402ebd12aabadf99fa6472443d2a7247e129033892999ba1R1879).

Reply:
Thanks! :)